library/agent/api-discovery/getApiInfo.ts

```diff
@@ -9,7 +9,7 @@ export type APISpec = {
   auth?: APIAuthType[];
 };
 
-export type APIBodyInfo = {
```

Comment:
wait why?

Reply:
Unused export that throws a warning in aslant